CryptoPrices/Sources/Styleguide/Sources/Styleguide/View+Extensions/View+RenderIf.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  View+RenderIf.swift
+//  Styleguide
+//
+//  Created by Doan Thieu on 05/01/2023.
+//
+
+import SwiftUI
+
+private struct RenderIfModifier: ViewModifier {
+
+    let condition: Bool
```

Comment:
As we have the initializer below, we should set this private access 🙏 
```suggestion
   private let condition: Bool
```

Reply:
Thank you for spotting out 🙏 , updated with https://github.com/nimblehq/nimble-crypto-ios/pull/88/commits/fa7584af1a0ab5e480fac76ff19766889e7c0081